cartridge.lua

```diff
@@ -216,6 +217,8 @@ local function cfg(opts, box_opts)
         console_sock = '?string',
         webui_blacklist = '?table',
         upgrade_schema = '?boolean',
+        probe_uri_timeout = '?number',
+        probe_uri_delay = '?number',
```

Comment:
Options variety increase complexity. I would recommend acquiring it with `argparse` and keeping it private.